exams/pearson/download_test.py

```diff
@@ -342,32 +346,29 @@ def test_process_result_vcdc(self):
         with patch('exams.pearson.download.VCDCReader.read', return_value=self.success_results):
             assert self.processor.process_vcdc_file("/tmp/file.ext") == (True, [])
 
-            for profile in self.success_profiles:
-                profile.refresh_from_db()
-                assert profile.status == ExamProfile.PROFILE_SUCCESS
+        for profile in self.success_profiles:
+            profile.refresh_from_db()
+            assert profile.status == ExamProfile.PROFILE_SUCCESS
 
     def test_process_result_vcdc_when_error(self):
         """
         Test situation where we get failure results back
         """
 
         with patch('exams.pearson.download.VCDCReader.read', return_value=self.all_results):
-            assert self.processor.process_vcdc_file("/tmp/file.ext") == (
-                True,
-                [
-                    "ExamProfile sync failed for user `{username}`.".format(
-                        username=self.failure_profiles[0].profile.user.username),
-                    "ExamProfile sync failed for user `{username}`. Received error: 'Bad address'.".format(
-                        username=self.failure_profiles[1].profile.user.username),
-                ]
-            )
-            for profile in self.success_profiles:
-                profile.refresh_from_db()
-                assert profile.status == ExamProfile.PROFILE_SUCCESS
+            result, errors = self.processor.process_vcdc_file("/tmp/file.ext")
+
+        assert result is True
+        assert all(error.startswith('ExamProfile sync failed:') for error in errors)
+        assert 'error=`Bad address`' in errors[1]
```

Comment:
Yes, that's the way they were originally formatted, I can change that if you think we should since I'm touching that code in this PR anyway.